toolbox/Dockerfile

```diff
@@ -27,8 +27,10 @@ RUN apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 08b
         gdb \
         iperf3 \
         jq \
+        kmod \
         less \
         man-db \
+        sudo \
```

Comment:
unfortunately yes, because `rook` under the covers shells out to `sudo` blindly: #295